packages/firestore-types/index.d.ts

```diff
@@ -216,6 +216,13 @@ export class FirebaseFirestore {
    */
   app: any;
 
+  /**
+   * Clears the persistence cache. This can only be called when the client is not running.
```

Comment:
A few thoughts on this comment:
  * In the `enablePersistence` comment above we say we "enable persistent storage". Let's describe this comparably.
  * We should also define "not running" concretely: see how `enablePersistence` defines the before state, but also after deleting the app.
  * We also need to adjust the comment for enablePersistence(), since `clearPersistence` is allowed.
  * You should indicate that we'll throw an exception if the client is still running
  * You should indicate under which conditions the promise can be rejected.

(And again, all in a separate PR.)